spinalcordtoolbox/scripts/sct_download_data.py

```diff
@@ -134,22 +134,19 @@
     },
     "deepseg_gm_models": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/deepseg_gm_models/releases/download/r20180205/20180205_deepseg_gm_models.zip",
-            "https://osf.io/b9y4x/?action=download",
+            "https://github.com/spinalcordtoolbox/deepseg_keras_onnx_conversion/releases/download/initial_build_opset_11/deepseg_gm_models_onnx.zip"
         ],
         "default_location": os.path.join(__sct_dir__, "data", "deepseg_gm_models"),
     },
     "deepseg_sc_models": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/deepseg_sc_models/releases/download/r20180610/20180610_deepseg_sc_models.zip",
-            "https://osf.io/avf97/?action=download",
+            "https://github.com/spinalcordtoolbox/deepseg_keras_onnx_conversion/releases/download/initial_build_opset_11/deepseg_sc_models_onnx.zip"
         ],
         "default_location": os.path.join(__sct_dir__, "data", "deepseg_sc_models"),
     },
     "deepseg_lesion_models": {
         "mirrors": [
-            "https://github.com/spinalcordtoolbox/deepseg_lesion_models/releases/download/r20180613/20180613_deepseg_lesion_models.zip",
-            "https://osf.io/eg7v9/?action=download",
+            "https://github.com/spinalcordtoolbox/deepseg_keras_onnx_conversion/releases/download/initial_build_opset_11/deepseg_lesion_models_onnx.zip"
```

Comment:
It might be better to fold the `.ipynb` file into `dev/`, and fold the model releases into their respective repos (`deepseg_gm_models`, `deepseg_sc_models`, `deepseg_lesion_models`), rather than having a dedicated repo for this.

Reply:
Done in [d11e680](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3738/commits/d11e680d73f10bd5e58723fd6ab3e4ae310c2b4f) and [3800dbf](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3738/commits/3800dbf0b626d3c54146135156f997f3ab911260).